pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -518,7 +512,10 @@ impl CustomScan for PdbScan {
             .get_reader()
             .expect("search index reader should have been constructed correctly");
 
-        state.custom_state_mut().query = Some(search_index.query(&search_config, &search_reader));
+        let search_query_input = &state.custom_state().search_query_input.clone();
+
+        state.custom_state_mut().query =
+            Some(search_index.query(&search_query_input, &search_reader));
```

Comment:
```suggestion
         state.custom_state_mut().query =
            Some(search_index.query(&state.custom_state().search_query_input, &search_reader));
```

to avoid the `.clone()`